packages/next/src/build/normalize-catchall-routes.test.ts

```diff
@@ -114,6 +114,7 @@ describe('normalizeCatchallRoutes', () => {
     }
 
     const initialAppPaths = JSON.parse(JSON.stringify(appPaths))
+    normalizeCatchAllRoutes(appPaths)
```

Comment:
```suggestion
    // ensure values are correct after normalizing
    normalizeCatchAllRoutes(appPaths)
```
Let's add a comment so it's more clear this is relevant.

Reply:
Actually, this function does the normalizing and it's the function after that does the check.

```
    // normalize appPaths against catchAlls
    normalizeCatchAllRoutes(appPaths)

    // ensure values are correct after normalizing
    ....
```

Reply:
@ijjk I have pushed https://github.com/vercel/next.js/pull/60777/commits/e9ef5b402f210fae652e0a613cf200a748b75f29 with

- revised comment (see above)
- added it to _all_ test cases within the file, not just the one that was restored